src/components/PeriodicReports/FinancialReportCheckList.tsx

```diff
@@ -0,0 +1,57 @@
+import { Checkbox, FormControl, FormControlLabel } from '@material-ui/core';
+import React from 'react';
+import {
+  FinancialReportsQuery,
+  NarrativeReportsQuery,
+} from '../../scenes/Projects/Reports/ProjectReports.generated';
+
+const ReportGuideAction = {
+  SET_RANGE: 'SET_RANGE',
+  ADD_MANAGING_PARTNER: 'ADD_MANAGING_PARTNER',
+  SET_REPORT_FREQUENCY: 'SET_REPORT_FREQUENCY',
+};
+
+export const FinancialReportCheckList = ({
```

Comment:
It still would be good to have these link to or open a dialog to take the action needed.
FWIW I used a [`<List>`](https://v4.mui.com/components/lists/#checkbox) component in my screenshot